question-service/routers/questions.py

```diff
@@ -15,6 +15,15 @@
     },
 )
 
+admin_router = APIRouter(
```

Comment:
I think going forward I can split this into two different files for admin and non-admin routes.